abodepy/devices/__init__.py

```diff
@@ -198,11 +198,6 @@ def status(self):
         """Shortcut to get the generic status of a device."""
         return self.get_value('status')
 
-    @property
-    def level(self):
```

Comment:
This property decorator was not being used and isn't actually pulling any data from the device JSON since it's looking for `'level'` and not `'level'` inside the `'statuses'` dictionary. Regardless, the functionality already exists with the `brightness` property decorator in the `light` module.